packages/expo-battery/src/Battery.ts

```diff
@@ -18,19 +18,21 @@ const BatteryEventEmitter = new EventEmitter(ExpoBattery);
  * Deprecated
  */
 export declare const isSupported: boolean;
-Object.defineProperties(module.exports, {
-  isSupported: {
-    enumerable: true,
-    get() {
-      deprecate('expo-battery', 'Battery.isSupported', {
-        replacement: 'Battery.isAvailableAsync',
-        currentVersion: require('../package.json').version,
-        versionToRemove: '3.0.0',
-      });
-      return (ExpoBattery && ExpoBattery.isSupported) || false;
+if (module.exports) {
```

Comment:
The only place I found `if (module.exports)` was 

https://github.com/expo/expo/blob/8076e32814eba2b07729c2b724d92186fc0ff206/packages/expo/src/Expo.fx.tsx#L50

Should we do exactly the same here?

Reply:
imho `module` may not be defined in some contexts and if we use JS strict mode then we should do:

`if (global.module && module.exports)`

without `global.` we could get `ReferenceError: module is not defined`

Reply:
fwiw prism uses just `module` to do the same thing
https://github.com/expo/expo/blob/98b9551b9eaf2258ac397362042244269ee09f3c/docs/static/libs/prism/prism.js#L332

Reply:
this is just another way to prevent reference error I mentioned 😛

Reply:
Note that in node `module` is not a property of `global` (it's injected into each module's scope as as argument), so a `typeof` check is a better idea

Reply:
that's fair point @SimenB, thanks 😉